tools/local-sensor/main.go

```diff
@@ -243,9 +250,24 @@ type sensorMessagesJSONOutput struct {
 	MessagesFromSensor []*central.MsgFromSensor `json:"messages_from_sensor"`
 }
 
-func dumpMessages(messages []*central.MsgFromSensor, start, end time.Time, outfile string) {
+func dumpMessages(messages []*central.MsgFromSensor, start, end time.Time, outfile string, binaryOutput bool) {
 	dateFormat := "02.01.15 11:06:39"
 	log.Printf("Dumping all sensor messages to file: %s\n", outfile)
+	if binaryOutput {
+		fname := strings.TrimSuffix(outfile, filepath.Ext(outfile))
+		file, err := os.OpenFile(fmt.Sprintf("%s.bin", fname), os.O_APPEND|os.O_CREATE|os.O_WRONLY, 0644)
```

Comment:
If `-central-out` is `result.bin` the two files would overwrite.

Reply:
Honestly, I didn't think someone would name a `json` file with a `.bin` extension 😅

Reply:
I wouldn't be surprised 😛